underscore.js

```diff
@@ -84,18 +84,23 @@
     };
   };
 
+  var builtinIteratee;
```

Comment:
@akre54, @jridgewell I'd be open to leaving this as is, switching the ESLint rule to a warning, or ignoring the warning with an inline comment. Our previous conversation got destroyed with my force-push.